app/client/src/components/pages/Community.Tabs.Monitoring.js

```diff
@@ -344,159 +404,74 @@ function SensorsTab({ usgsStreamgagesDisplayed, setUsgsStreamgagesDisplayed }) {
 
   const services = useServicesContext();
 
-  const { usgsStreamgagesLayer, watershed } = useContext(LocationSearchContext);
+  const { watershed } = useContext(LocationSearchContext);
 
-  const [normalizedUsgsStreamgages, setNormalizedUsgsStreamgages] = useState(
-    [],
+  const normalizedUsgsStreamgages = useStreamgageData(
+    usgsStreamgages.data,
+    usgsStreamgages.status,
```

Comment:
Can you update this to just pass in the entire `usgsStreamgages` object instead of passing in the data and status separately?